exonum/src/api/node/public/explorer.rs

```diff
@@ -171,6 +172,80 @@ impl AsRef<[u8]> for TransactionHex {
     }
 }
 
+/// Status of a call.
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct CallStatusResponse {
+    /// Call status
+    pub status: ExecutionStatus,
+    /// Call execution proof
+    pub call_proof: MapProof<CallInBlock, ExecutionError>,
+}
+
+/// Call status query parameters.
+#[derive(Debug, Clone)]
+#[derive(Serialize, Deserialize)]
+#[serde(untagged)]
+pub enum CallStatusQuery {
```

Comment:
Since this query isn't obvious, I'd expect a doc-comment for it to contain an example of the expected `json`. It will be nice to have an example of output `json` in doc-comment for endpoint.